dags/veda_data_pipeline/veda_vector_pipeline.py

```diff
@@ -69,25 +69,23 @@ def ingest_vector_task(payload):
     return handler(payload_src=payload, vector_secret_name=vector_secret_name,
                    assume_role_arn=read_role_arn)
 
-def get_ingest_vector_dag(id, event={}):
-    
-    params_dag_run_conf = event or template_dag_run_conf
+def get_ingest_vector_dag(*, id: str, event: dict):
 
     with DAG(
             id,
-            schedule_interval=params_dag_run_conf.get("schedule", None),
-            params=params_dag_run_conf, 
+            schedule_interval=event.get("schedule", None),
+            params=event, 
             **dag_args
         ) as dag:
         start = DummyOperator(task_id="Start", dag=dag)
         end = DummyOperator(task_id="End", trigger_rule=TriggerRule.ONE_SUCCESS, dag=dag)
-        discover = discover_from_s3_task()
+        discover = discover_from_s3_task(event)
```

Comment:
Making the params explicit here made it easier to see why the scheduled DAG was failing without an event but the default user-triggered DAG was working (because the run config is submitted when starting a manual run)